sass/components/Dashboard/capacity.scss

```diff
@@ -1,13 +1,16 @@
 .kubevirt-capacity__items {
   display: flex;
-  justify-content: space-between;
+  justify-content: space-evenly;
   flex-wrap: wrap;
 }
 
 .kubevirt-capacity__item {
```

Comment:
the donuts are overflowing outside the card

Reply:
and the items are all jumbled together on smaller devices

Reply:
\+ the graphs of all items should be centered similarly

Reply:
The issue is happening in cosmos playground - i think that we need to import base.css from patternfly-react but them Im facing issue with webpack not being able to load fonts..

Reply:
can we fix it so it doesn't break without that css?